tests/data/regional_client_and_bucket/ci/taskcat.yml

```diff
@@ -0,0 +1,34 @@
+project:
+  owner: quickstart-eng@amazon.com
+  name: complex-auth-config-file
+  regions:
+    - ap-northeast-1
+    - ap-northeast-2
+    - ap-south-1
+    - ap-southeast-1
+    - ap-southeast-2
+    - ca-central-1
+    - eu-central-1
+    - eu-west-1
+    - eu-west-2
+    - us-east-1
+    - us-east-2
+    - us-west-1
+    - us-west-2
+    - sa-east-1
+  auth:
+    us-east-1: prod
+    sa-east-1: south-america-prod
+    us-east-2: staging
+tests:
+#  json-test:
```

Comment:
If we're not going to use this test can we remove it and it's associated files from the pr

Reply:
on 2nd thoughts I think we need this to be re-enabled. without it we're not looking at a broad enough set of possible overrides.